lib/heroku/command/domains.rb

```diff
@@ -13,17 +14,35 @@ class Domains < Base
     #Examples:
     #
     # $ heroku domains
-    # === Domain names for example
-    # example.com
+    # === Development Domain
+    # example.herokuapp.com
+    #
+    # === Custom Domains
+    # Domain Name  CNAME Target
```

Comment:
@jesperfj, @itchymutt: are you ok with using `DNS Record Target` both here and Dashboard?